main/src/main/java/org/mobilitydata/gtfsvalidator/notice/DecreasingShapeDistanceNotice.java

```diff
@@ -0,0 +1,24 @@
+package org.mobilitydata.gtfsvalidator.notice;
+
+import com.google.common.collect.ImmutableMap;
+
+public class DecreasingShapeDistanceNotice extends Notice {
```

Comment:
For consistency with the other notices, I suggest reming this one to `DecreasingShapeDistance`

Reply:
Another solution would be to append the term "notice" to the other notices:
- `InconsistentAgencyField`
- `StartAndEndDateOutOfOrder`
- `StopTimeWithArrivalBeforePreviousDepartureTime`
- `StopTimeWithDepartureBeforeArrivalTime`
- `StopTimeWithOnlyArrivalOrDepartureTime`
- `UnusedShape`

Reply:
Notices look similar to Java exceptions. Since exceptions have a suffix (such as child classes here: https://docs.oracle.com/javase/7/docs/api/java/lang/RuntimeException.html), I decided to add a suffix to notice classes.